src/System.Private.CoreLib/System.Private.CoreLib.csproj

```diff
@@ -372,7 +372,6 @@
     <Compile Include="$(BclSourcesRoot)\System\Reflection\Metadata\AssemblyExtensions.cs" />
   </ItemGroup>
   <ItemGroup>
-    <Compile Include="$(BclSourcesRoot)\System\Globalization\CharUnicodeInfoData.cs" />
```

Comment:
Could you please delete this file as well?

Reply:
interesting, I did move command in PS windows and I thought it deleted the original file but it didn't. anyway, I'll fix that. Thanks for the catch.